sdk/core/azure-core/inc/azure/core/credentials/token_credential_options.hpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Options for #Azure::Core::Credentials::TokenCredential.
+ */
+
+#pragma once
+
+#include "azure/core/internal/client_options.hpp"
+
+namespace Azure { namespace Core { namespace Credentials {
+  /**
+   * @brief Defines options for #Azure::Core::Credentials::TokenCredential.
+   */
+  struct TokenCredentialOptions : public Azure::Core::_internal::ClientOptions
```

Comment:
Needs CL entry.

Reply:
It is not a breaking change, and has the default value. All the code you could compile before can still compile now.